api/auth/index.ts

```diff
@@ -67,7 +67,7 @@ export const getXuiNodeMiddleware = () => {
         callbackURL: getConfigValue(SERVICES_IDAM_OAUTH_CALLBACK_URL),
         clientID: idamClient,
         clientSecret: secret,
-        discoveryEndpoint: `${idamWebUrl}/o`,
+        discoveryEndpoint: `${idamWebUrl}/o/.well-known/openid-configuration`,
```

Comment:
directories that start with a . are generally hidden/system dirs. Are we sure we are supposed to hard code it this way?

Reply:
it's not a directory, but the standard for autodiscovery endpoints @see https://ldapwiki.com/wiki/Openid-configuration#:~:text=Openid%2Dconfiguration%20is%20a%20Well,the%20Identity%20Provider%20(IDP).

Reply:
However I do agree that perhaps it shouldn't be hardcoded, maybe a task in future to move this into the config instead

Reply:
its inline with guidelines, not a show stopper but a good call to query it.